docs/resources/Channel.md

```diff
@@ -656,28 +656,18 @@ Deletes all reactions on a message. This endpoint requires the 'MANAGE_MESSAGES'
 
 ## Edit Message % PATCH /channels/{channel.id#DOCS_RESOURCES_CHANNEL/channel-object}/messages/{message.id#DOCS_RESOURCES_CHANNEL/message-object}
 
-Edit a previously sent message. You can only edit messages that have been sent by the current user. Returns a [message](#DOCS_RESOURCES_CHANNEL/message-object) object. Fires a [Message Update](#DOCS_TOPICS_GATEWAY/message-update) Gateway event.
+Edit a previously sent message. You can edit message `content`, `embed`, and `flags` if `author` is current user. Otherwise only `flags` can be edited with `MANAGE_MESSAGES` permission in the channel. Returns a [message](#DOCS_RESOURCES_CHANNEL/message-object) object. Fires a [Message Update](#DOCS_TOPICS_GATEWAY/message-update) Gateway event.
 
 > info
 > All parameters to this endpoint are optional.
 
 ###### JSON Params
 
-| Field   | Type                                                 | Description                                      |
-| ------- | ---------------------------------------------------- | ------------------------------------------------ |
-| content | string                                               | the new message contents (up to 2000 characters) |
-| embed   | [embed](#DOCS_RESOURCES_CHANNEL/embed-object) object | embedded `rich` content                          |
-
-## Suppress Message Embeds % POST /channels/{channel.id#DOCS_RESOURCES_CHANNEL/channel-object}/messages/{message.id#DOCS_RESOURCES_CHANNEL/message-object}/suppress-embeds
```

Comment:
Why was this endpoint removed? It seems good to be able to update only this flag without having to first fetch the message.